Cargo.toml

```diff
@@ -10,9 +10,11 @@ syn = { version = "2.0.85", features = ["full"] }
 quote = "1.0.37"
 proc-macro2 = "1.0.89"
 proc-macro-error2 = "2.0.1"
-llvm-sys = { version = "191" }
+llvm-sys = { version = "191", features = ["force-static"] }
```

Comment:
I get this is slightly out of scope but honestly it's too small to get its own PR imo.

This forces LLVM to be linked statically into our binaries instead of dynamically using a
separate file. Just makes distribution easier.

Reply:
Could you please measure how badly this affects binary sizes

Reply:
> Could you please measure how badly this affects binary sizes

It shouldn't affect binary sizes much but even if it'd still have to be included even if in just a separate file so it doesn't change much if we do it dynamically else than less convenience.